src/libraries/sendtohelixhelp-mobile-non-wasm.proj

```diff
@@ -0,0 +1,143 @@
+<Project>
```

Comment:
The name seems pretty complicated, `wasm` is not mobile per se, so should this file name only be `mobile`?

Reply:
I agree that the name is complicated. `$(TargetsMobile)` includes the browser, so I had this. @steveisok would `sendtohelixhelp-mobile.proj` be fine?

Reply:
yeah, I think mobile is fine.